### DISCUSSION TITLE
Can't use an undefined value as a HASH reference

### DISCUSSION THREAD

**Discussion by frankrex** [Difegue/LANraragi#926] [Q&A]
**LRR Version and OS**
Docker nightly
**Bug Details**
Hello, thank you for your awesome project.

I recently deployed the nightly version of LANraragi docker image on synology and placed some comic zip files under content dir.

In the LANraragi plugin config, ehentai's `ipb_member_id` and `ipb_pass_hash` are set, but an error is reported when obtaining metadata:

```
Error: Can't use an undefined value as a HASH reference at /home/koyomi/lanraragi/script/../lib/LANraragi/Plugin/Metadata/EHentai.pm line 259.
```

But I can't see any error log in minio console.

**Matching Logs**

None

**Screenshots**

![error](https://github.com/Difegue/LANraragi/assets/13638916/c49a1145-6b34-4748-b0ca-133f2c638b7e)

**Comment by Difegue** [comment]
Looks like you're not getting valid data from EH.  
Debug mode would give you the full HTML responses in plugin logs which should be helpful

**Comment by frankrex** [comment]
Thank you for your help. It seems that the IP used is temporarily blocked by EH and will be restored after a while :)
